lib/plugins/stats.js

```diff
@@ -1,3 +1,5 @@
+const axios = require('axios')
```

Comment:
We should have a chat about what we want to do for GraphQL support in Probot before committing to this. I'd love to pick a GraphQL client and pull it in pretty soon. cc @gr2m

Reply:
I just found this as well: [graphql-request](https://github.com/graphcool/graphql-request)

Seems like it matches our use case very well. Async, Typescript support, and a small library. I found it because I was looking at one of the graphql examples from [micro](https://github.com/zeit/micro/blob/master/examples/with-graphql-request/index.js).

Reply:
graphql-request looks great